bin/adam-shell

```diff
@@ -28,8 +26,34 @@ if [[ -z $@ && -n "$ADAM_OPTS" ]]; then
     echo "Run adam-shell instead as adam-shell <spark-args>"
 fi
 
+# Find ADAM cli assembly jar
+ADAM_CLI_JAR=
+if [ -f "$SCRIPT_DIR/repo" ]; then
```

Comment:
I updated the distribution assembly.xml to copy the adam-cli jar to repo, so this check is to see if the user is trying to run from the dist.  The Spark build uses a separate assembly module and has a more involved find (in the ```spark-class``` script), we may wish to crib from there.